Overview.src.html

```diff
@@ -4630,10 +4638,14 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
  <span title=concept-request-cache-mode>cache mode</span>,
  <span title=concept-request-redirect-mode>redirect mode</span> is
  <var>request</var>'s
- <span title=concept-request-redirect-mode>redirect mode</span>, and
+ <span title=concept-request-redirect-mode>redirect mode</span>,
  <span title=concept-request-integrity-metadata>integrity metadata</span> is
  <var>request</var>'s
- <span title=concept-request-integrity-metadata>integrity metadata</span>.
+ <span title=concept-request-integrity-metadata>integrity metadata</span>, and
+ <span title=concept-request-keep-alive-flag>keep-alive flag</span> is
```

Comment:
> <p>A <span title=concept-request>request</span> has an associated
<dfn>keep-alive flag</dfn>. Unless stated otherwise it is unset.

I think that should do the trick?